coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/HiveDbSchema.java

```diff
@@ -83,7 +83,7 @@ public RelProtoDataType getType(String s) {
 
   /**
    * A Hive DB does not have subschema
-   * @param name
+   * @param name Subscheme name
```

Comment:
Subshema

Reply:
I am assuming you mean 'Subschema'. :) Fixed.